src/app/code/community/Shopgate/Cloudapi/Model/Api2/Observers/WishlistsItemsRetrieve.php

```diff
@@ -49,4 +50,21 @@ public function execute(Varien_Event_Observer $observer)
         $output = $observer->getData('output');
         $output->setData('items', $items);
     }
+
+    /**
+     * @param Mage_Wishlist_Model_Item $item
+     */
+    private function appendChildId($item)
+    {
+        $buyRequestItem = new Varien_Object($item->getBuyRequest()->getData());
```

Comment:
isn't the buyRequest a VarienObject if you are calling getData() on it?

Reply:
^^ - you are right